### ISSUE TITLE
One-stop font color (theme) tweak-through-app-layers feasibility at config.json

### ISSUE THREAD

**Issue by kazuf3** [microsoft/terminal#14207]
# Description of the new feature/enhancement

TL; DR
Make color theme tweaks one-stop at `config.json`.

Currently there are many choices where user make the font color selection in Terminal.
For example, an user may pick font color in their CUI text editor, shell config, Tmux config or Terminal config.
Let's say a user picks solarized dark theme in one's `config.json`, but his different layers of apps overwrite them respectively, so one won't see solarized dark theme in his editor.

# Proposed technical implementation details (optional)

Pass current color theme variables from `config.json` into corresponding environment variables to WSL2.

**Comment by carlos-zamora** [comment]
I could be wrong, but I don't think there's a reasonable way to accomplish this. Windows Terminal lets you define the 16 colors in the color table, right? So if the connected shell requests "black" or "bright black", I assume the shell resolves it using the color table we provided and outputs the text in the desired color.

Now, if the shell outputs text in a specific RGB format, we can't really get in the way of that. Tmux, for example, would probably require some kind of initialization such that the color scheme from the Windows Terminal profile is converted and injected into Tmux's config file so that it uses that.

To make things worse, if there's one tmux.exe, how would this work with multiple WT profiles running tmux across different panes/tabs?

I like the idea and appreciate the feedback request, but I'm going to close this as not planned. Happy to reopen it if I'm wrong though (totally possible).

**Comment by kazuf3** [comment]
@carlos-zamora 

gotcha. 

Could you confirm that there are no way/plans to pass environment variables from `config.json` to WSL2 guest, not limited to this use case? At least it would help pass color scheme to editors inside of WSL2 guest.
